test/endpoints/test_get_record_by_id.py

```diff
@@ -52,19 +52,19 @@ def test_valid_get_record_by_id(
             pytest.param(
                 "20220408132830",
                 {
-                    "N_COMP_FF_IMAGE": "2fa3239b5f46b110964f02a2e6c6dd62",
-                    "N_COMP_NF_IMAGE": "16624db9b93bfe8176cd275d24522bc5",
-                    "N_LEG1_GREEN_NF_IMAGE": "1fcf716b4f670dc4a901ca77ea0b8ee5",
+                    "N_COMP_FF_IMAGE": "8c997363cc8c3373",
+                    "N_COMP_NF_IMAGE": "c2c63f8f783c3138",
+                    "N_LEG1_GREEN_NF_IMAGE": "8785789c6a7a6761",
```

Comment:
The parameterised variable for this is called `expected_thumbnail_md5s`. Can you change it something like `expected_thumbnails_hashes` as we no longer use MD5 checksums please? `expected_thumbnail_md5s` is used across multiple files so you can do a search and replace in VSCode to change all instances in all files.

There's a few other instances of MD5 still being mentioned (e.g. in variable names and docstrings). Could you search for `md5` across the codebase and replace mention of MD5 with the type of hashing we use now please?